### PR TITLE
Fixed #32647 -- Restored multi-row select with shift-modifier in admin changelist.

### PR DESCRIPTION
**Pull request by carltongibson**

### CODE REVIEW

**Comment by carltongibson** [comment]
buildbot, test on selenium.

File: docs/releases/3.2.1.txt [hunk 1]
```diff
@@ -44,3 +44,6 @@ Bugfixes
 * Fixed a regression in Django 3.2 that caused a crash when decoding a cookie
   value, used by ``django.contrib.messages.storage.cookie.CookieStorage``, in
   the pre-Django 3.2 format (:ticket:`32643`).
+
+* Fixed a regression in Django 3.2 when using the shift-key modifier to select
```
Comment (felixxm) [inline docs/releases/3.2.1.txt:48]: Maybe:
```suggestion
* Fixed a regression in Django 3.2 that stopped the shift-key modifier to select
```

**Review by felixxm — APPROVED** [review]
@carltongibson Thanks :+1:

**Comment by carltongibson** [comment]
buildbot, test on selenium.

**Pull request merged**